### DISCUSSION TITLE
Multi Document YAML Files

### DISCUSSION THREAD

**Discussion by ghost** [stoplightio/spectral#2516] [Q&A]
I am investigating using Spectral Lint to help check YAML files for 'correctness'.
However the input YAML Files are 'multi document' - i.e separated by `---`

When processing I get this error:
```
 1:1  error  parser  Expected a single document in the stream, but found more
```
From the documents I cant find a way to support scanning the entire file (and the multiple documents within).
Is it possible?

**Comment by P0lip** [comment]
Hey!
Unfortunately, that is not possible at the moment. :(

**Reply by lexfrei** [reply]
Any news on this? It's really annoying and forced me to disable the vscode extension.

**Comment by vordimous** [comment]
I am getting this error.
